File: Chapter 1/Complete Pack/f.c
#include <stdio.h>

int main(int argc, char *args[])
{
	int B, C;
	printf("Enter B : ");
	scanf("%d", &B);

	printf("Enter C : ");
	scanf("%d", &C);

	B = B - C;
	C = C + B; //gets B's value
	B = C - B; //gets C's value

	printf("After Swapping Value of B is : %d \n", B);
	printf("After Swapping Value of A is : %d \n", C);
	return 0;
}

File: Chapter 1/(f)/f.java
import java.util.Scanner;

class f
{
	static public void main(String args[])
	{
		int B,C;
		Scanner input=new Scanner(System.in);
		System.out.print("Enter B : ");
		B=input.nextInt();
		System.out.print("Enter C : ");
		C=input.nextInt();
		B=B-C;
		C=C+B;
		B=C-B;
		System.out.println("After Swapping B is : "+B);
		System.out.println("After Swapping C is : "+C);

	}
}